server/app/routes/users.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import {
+  authNewUser,
+  validate
+} from './../middlewares/auth';
+import {
+  newUsers,
+  logInUser
+} from './../middlewares/users';
+
+const userRoutes = express.Router();
+
+userRoutes.post('/auth/signup', authNewUser, newUsers);
+userRoutes.post('/auth/login', validate, logInUser);
+
+
+export default userRoutes;
```

Comment:
The default export should match the name of the file. Please change the name of the file to `userRoutes`

Reply:
There are other instances, please correct them and pay attention to the little details.